org/views.py

```diff
@@ -51,4 +52,48 @@ def post(self,request):
             return Response({}, status.HTTP_201_CREATED)
         else:
             data = serializer.errors
-            return Response(data, status.HTTP_400_BAD_REQUEST)
\ No newline at end of file
+            return Response(data, status.HTTP_400_BAD_REQUEST)
+
+
+@swagger_auto_schema(
+    operation_id="add_volunteer",
+    operation_description="When an authenticated user hits this API it gets added to the volunteer group",
+    method='get',
+    responses={
+        '201': set_example(responses.add_volunteer_201),
+        '400': set_example(responses.org_not_present_400),
+        '401': set_example(responses.user_unauthorized_401),
+        '200': set_example(responses.user_already_present_200)
```

Comment:
```suggestion
        '409': set_example(responses.user_already_present_200)
```
 I guess 409 conflict will be more appropriate :thinking: . What u think?